lib/go/internal/v0/req_context.go

```diff
@@ -54,7 +51,7 @@ func (e *DataSource) AuthenticationToken() *ecdata.AuthenticationToken {
 		if token, err := e.impl.ValidateToken(e.raw.AuthToken); err != nil {
 			// empty jwt token is considered "normal", no need to spam them in logs.
 			if !errors.Is(err, ErrEmptyToken) {
-				e.Log(e.getCtx(), "token validation failed: "+err.Error())
+				slog.ErrorContext(e.getCtx(), "token validation failed: "+err.Error())
```

Comment:
since we are using slog now I'd prefer to log in kv pairs instead:
```suggestion
				slog.ErrorContext(e.getCtx(), "token validation failed", "err", err)
```
same for the other places we are switching to slog.